docs/learn/data-availability.md

```diff
@@ -0,0 +1,35 @@
+# Data Availability
+This page describes the data persistence and data availability model for [streams](./glossary.md#streams) on Ceramic.
+
+## **Overview**
+Stream data can be divided into two main categories: [commits](./glossary.md#commits) and [state](./glossary.md#state). While there is some overlap, these two types of data have enough difference that they should be considered separately when planning how to persist and host your streams. All commits and state for a given stream must be available when performing [writes](../build/writes.md) or [queries](../build/queries.md) on that stream; if not, your client will return a `50X` error. This error will disappear once all data comes back online. In the event that one or more commits for the given stream were permanently lost due to improper persistence management, then this stream will be corrupted and the error will not disappear.
+
+## **Stream commits**
+Every stream is an event log consisting of one of more commits, and each commit contains of at least two [IPLD](./glossary.md#ipld) objects. Collectively these commits store the data that makes up the content of a stream.
+
+### Caching
+Ceramic nodes have a built-in caching mechanism for short-term storage of commits. Whenever a Ceramic node performs a [write](../build/writes.md) or a [query](../build/queries.md) on a stream, all commits for that stream are first synced from the [network](./glossary.md#networks) and automatically loaded into that node's in-memory cache. This results in the most popular streams being replicated the most, providing some level of data persistence and availability. However to preserve disk space and node resources, in-memory cache is currently limited to 50 streams. Once that number is reached, the oldest streams will be evicted from the node's cache in order to make room for newer ones. 
```

Comment:
Where did the number 50 come from? By default the cache size is 500, but it's also configurable per-node

Reply:
Ah I thought it was either 50 or 500 from our convo the other day. And good to know. Will update this to reflect this info.